src/common/chip/chip.module.scss

```diff
@@ -0,0 +1,35 @@
+@import 'spacings';
+
+.chip {
+  --background-color: transparent;
+  --color: default;
+
+  background-color: var(--background-color);
+  color: var(--color);
+  padding: units(0.25) units(0.75);
+
+  &.grey {
+    --background-color: hsl(0, 0%, 90%);
+    --color: hsl(0, 0%, 30%);
+  }
+
+  &.green {
+    --background-color: hsl(174, 49%, 92%);
```

Comment:
are those random values (your own definition)?

Reply:
I took them from our layouts, I think it would be better to move them to our `colours` module and import them from there.